plugins/org.jkiss.dbeaver.ui.statistics/src/org/jkiss/dbeaver/ui/statistics/UIStatisticsActivator.java

```diff
@@ -53,21 +51,6 @@ public static void setTrackingEnabled(boolean enabled) {
             return;
         }
         setPreferenceValue(PREF_FEATURE_TRACKING_ENABLED, enabled);
-        if (tracker != null) {
```

Comment:
isTrackingEnabled must return true if we are in EAP mode

Reply:
The application instance is not available at this point. During the application startup, the preference is set to true, effectively emulating the desired behavior.